guide/designing-products/personal-finance.md

```diff
@@ -36,15 +36,11 @@ Illustration sources
 # Personal finance
 {:.no_toc}
 
-**If we consider Bitcoin a new form of money, we should consider how we interact with existing forms of money and how Bitcoin compares. This can guide us towards behavioral patterns that users may already be familiar with.**
+**If we consider Bitcoin a new form of money, it’s only natural that we compare it to existing forms of currency.This section views Bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making.**
 
-Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.
+This section views bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making. Because Bitcoin is both different and new, we can’t depend on solutions to it that worked for conventional financial products.
```

Comment:
This paragraph (starting with "This section") is now duplicated. Could you please double-check and ensure the correct text is being added and removed here and in the surrounding paragraphs?

Reply:
oops nice catch fixed https://github.com/BitcoinDesign/Guide/pull/374/commits/a9be74dce4bc3cbf5be164e4504eb70b01ca63d5